integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -160,7 +160,7 @@ case class CarbonLoadDataCommand(
         if (isOverwriteTable) {
           LOGGER.info(s"Overwrite of carbon table with $dbName.$tableName is in progress")
         }
-        if (carbonLoadModel.getLoadMetadataDetails.isEmpty && carbonLoadModel.getUseOnePass &&
+        if (carbonLoadModel.getLoadMetadataDetails.size() == 1 && carbonLoadModel.getUseOnePass &&
```

Comment:
This if condition can be remove as the high_cardinality check has been removed. 
@manishgupta88 Please suggest